index.html

```diff
@@ -8,13 +8,15 @@
     <link href="https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="styles.css">
     <script src="https://kit.fontawesome.com/594a9a3356.js" crossorigin="anonymous"></script>
+    <script src="index.js"></script>
     <title>Practise</title>
 </head>
 <body>
     <header>
         <nav class="navbar">
+            <span class="navbar__logo"><a class="navbar__link" href="#"><i class="fa-brands fa-uikit"></i></a></span>
             <ul class="navbar__list">
-                <li class="navbar__item"><a class="navbar__link" href="#"><i class="fa-brands fa-uikit"></i></a></li>
+                <li class="navbar__item navbar__item--logo"><a class="navbar__link" href="#"><i class="fa-brands fa-uikit"></i></a></li>
```

Comment:
is it class added on some hover? why it has visibility change, not the display? is it intended?

Reply:
This class is added as a modifier for the positioning purpose. I used that as a solution regarding different behaviour of mobile and desktop logos.

Good point with visibility. It was not intended, I will change it for a display.